### ISSUE TITLE
Backup exposure history to transfer to new/reseted phone

### ISSUE THREAD

**Issue by ma6** [corona-warn-app/cwa-app-android#628]
## Feature description

Create a backup in order to be able to continue using the app without exposure data loss when moving to a new phone. Coming with that restoring the exposure data on a new phone.

## Problem and motivation

When a user changes their phone all exposure data for the last two weeks is lost. If they can make a backup of the data and restore it on their new phone this data could still be accessible and they could be warned and warn others about the infection. 

## Is this something you're interested in working on

Yes, from UX POV.

**Comment by corneliusroemer** [comment]
Good idea - but may be impossible to do since the important bits are within the closed GAP API. CWA has not access to this, can't download keys. So this will most likely not work.

A limited workaround is to keep the old phone active and checking for exposure of any encounters hitherto. So at least you may get notified more or less correctly about past encounters.

**Comment by gizmo21** [comment]
Well and "keeping active" means lay it in the drawer for 14 days with WiFi only (no SIM needed) and charge it once a day. So could only be difficult with a work mobile.

**Comment by tkowark** [comment]
Dear @ma6 ,

thank you very much for your proposal.

To give the community a space for discussing new feature ideas or enhancements that might not be taken up immediately by the development teams, we created the cwa-wishlist repository.

This idea is one of these feature ideas or enhancements. Consequently, we will move this issue to this repository and allow the community and you to further refine it, discuss pros and cons, and evaluate alternatives.

The issue will not be closed in this repository to ensure long-term visibility. All issues in the cwa-wishlist repository will be discussed with our contracting entities. They will decide whether they will be implemented or pursued further. As soon as we have any updates, we will let you know about the details.

Best regards,
TK
Corona-Warn-App Open Source Team
